src/Internal/EventStoreCatchUpSubscription.php

```diff
@@ -163,37 +162,19 @@ private function loadHistoricalEvents(): void
     private function subscribeToStream(): void
     {
         if (! $this->shouldStop) {
-            $eventAppeared = new class(Closure::fromCallable([$this, 'enqueuePushedEvent'])) implements EventAppearedOnSubscription {
-                private $callback;
-
-                public function __construct(callable $callback)
-                {
-                    $this->callback = $callback;
-                }
-
-                public function __invoke(
-                    EventStoreSubscription $subscription,
-                    ResolvedEvent $resolvedEvent
-                ): void {
-                    ($this->callback)($subscription, $resolvedEvent);
-                }
+            $eventAppeared = function (
+                EventStoreSubscription $subscription,
+                ResolvedEvent $resolvedEvent
+            ): void {
+                ($this->callback)($subscription, $resolvedEvent);
             };
 
-            $subscriptionDropped = new class(Closure::fromCallable([$this, 'serverSubscriptionDropped'])) implements SubscriptionDropped {
-                private $callback;
-
-                public function __construct(callable $callback)
-                {
-                    $this->callback = $callback;
-                }
-
-                public function __invoke(
-                    EventStoreSubscription $subscription,
-                    SubscriptionDropReason $reason,
-                    ?Throwable $exception = null
-                ): void {
-                    ($this->callback)($reason, $exception);
-                }
+            $subscriptionDropped = function (
+                EventStoreSubscription $subscription,
+                SubscriptionDropReason $reason,
+                ?Throwable $exception = null
+            ): void {
+                ($this->callback)($reason, $exception);
```

Comment:
Same here, what is `$this->callback` ?

Reply:
You tell me. It was there before. :) Not sure from where it came from and how to substitute it now.